components/j-space-ui.tsx

```
@@ -0,0 +1,146 @@
+        }}>
+          <div className="window-body" style={{ overflowY: 'auto', height: 'calc(100% - 40px)' }}>
+            {content.projects.map((project, i) => (
+              <div key={i} style={{ marginBottom: '20px', borderBottom: '1px solid #808080', paddingBottom: '10px' }}>
+                <h3>{project.title}</h3>
+                <p>{project.description}</p>
+                {project.image && (
+                  <img src={getImageUrl(`${project.image}.png`)} alt={project.title} style={{ width: '100%', border: '1px solid #808080' }} 
+                       onError={(e) => { (e.target as HTMLImageElement).src = getImageUrl(`${project.image}.gif`) }} />
+                )}
+                <div style={{ marginTop: '10px' }}>
+                  {project.github && <a href={project.github} target="_blank" rel="noreferrer"><button>GitHub</button></a>}
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)